autopilot/contractor.go

```diff
@@ -1425,24 +1420,18 @@ func (c *contractor) refreshContract(ctx context.Context, w Worker, ci contractI
 	expectedStorage := renterFundsToExpectedStorage(renterFunds, contract.EndHeight()-cs.BlockHeight, ci.priceTable)
 	unallocatedCollateral := rev.MissedHostPayout().Sub(contract.ContractPrice)
 
-	// calculate the expected new collateral to determine the minNewCollateral.
-	// If the contract isn't below the min collateral, we don't enforce a
-	// minimum.
-	var minNewColl types.Currency
-	_, _, expectedNewCollateral := rhpv3.RenewalCosts(contract.Revision.FileContract, ci.priceTable, expectedStorage, contract.EndHeight())
-	if isBelowCollateralThreshold(expectedNewCollateral, unallocatedCollateral) {
-		minNewColl = minNewCollateral(unallocatedCollateral)
-	}
+	// a refresh should always result in a contract that has enough collateral
+	minNewCollateral := minRemainingCollateral(state.cfg, state.rs, renterFunds, settings, ci.priceTable).Mul64(2)
```

Comment:
is this `.Mul64(2)` a debug leftover?

Reply:
No, it's just a leeway to make sure we are not right at min but slightly above it.